common-util/api/tokenomics.js

```diff
@@ -0,0 +1,36 @@
+import { TOKEN_HOLDER_NETWORKS } from 'common-util/constants';
+import { TOKENOMICS_GRAPH_CLIENTS } from 'common-util/graphql/client';
+import { holderCountsQuery } from 'common-util/graphql/queries';
+
+const toLowerCaseAddress = (address) => address.toLowerCase();
+
+const fetchHolderCount = ({ key, token }) => {
+  const client = TOKENOMICS_GRAPH_CLIENTS[key];
+  if (!client) {
+    return Promise.resolve(0);
+  }
+
+  return client
+    .request(holderCountsQuery, { tokenId: toLowerCaseAddress(token) })
```

Comment:
we shouldn't lowercase address if the type is Bytes in subgraph, checksummed address should work as well. could you please check?